packages/shared/src/contexts/common/Confirm/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { ReactNode } from 'react'
+
+export interface ConfirmOptions {
+    title?: string
+    content: ReactNode | string
+    confirmLabel?: string
+    onSubmit?(result: boolean | null): void
```

Comment:
In this case, I suggest using treat `close dialog` as `cancel`, which means they should both be represented by `false`.

Reply:
Closing the dialog means "no data input", then `null` make sense.

In general, a `confirm` dialog should provides a cancel button for user input, however, without close button.


<img width="448" alt="image" src="https://user-images.githubusercontent.com/1141198/169934409-faa2f734-eb86-44c4-8618-5c45943b6f57.png">

And we do have such design, but it has not yet been implemented.
We will implement it as the final `Confirm` dialog.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/1141198/169934384-f49ff47a-39a8-4969-9e04-c6e0646877e2.png">